packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -682,11 +683,11 @@ export default function useAutocomplete(props) {
       // only have an opinion about this when closed
       'aria-activedescendant': popupOpen ? undefined : null,
       'aria-autocomplete': autoComplete ? 'both' : 'list',
-      'aria-controls': `${id}-popup`,
-      // autoComplete: 'off', // Disable browser's suggestion that might overlap with the popup.
-      autoComplete: 'disabled', // disable autocomplete and autofill
+      'aria-controls': popupOpen ? `${id}-popup` : null,
```

Comment:
Fix https://validator.w3.org/ error were aria-controls points to a missing element.